ORStools/gui/ORStoolsDialogConfig.py

```diff
@@ -149,13 +149,13 @@ def _remove_provider(self):
             provider_id = providers.index(provider)
             del self.temp_config["providers"][provider_id]
 
-    def _collapse_boxes(self):
+    def _collapse_boxes(self) -> None:
         """Collapse all QgsCollapsibleGroupBoxes."""
         collapsible_boxes = self.providers.findChildren(QgsCollapsibleGroupBox)
         for box in collapsible_boxes:
             box.setCollapsed(True)
 
-    def _add_box(self, name, url, key, timeout, new=False):
+    def _add_box(self, name, url, key, timeout, new=False) -> None:
```

Comment:
```suggestion
    def _add_box(self, name, url, key, timeout, new=False) -> None:
```

These arguments are missing hints

Reply:
Will do